### PR TITLE
test: port multiple_rows.js to Elixir test suite

### PR DESCRIPTION
**Pull request by dottorblaster**
## Overview

I just ported `multiple_rows.js` to the Elixir test suite.

## Testing recommendations

Just issue `make elixir`

## Related Issues or Pull Requests

none

## Checklist

- [X] Code is written and works correctly;
- [X] Changes are covered by tests;
- [X] Documentation reflects the changes;

### CODE REVIEW

**Comment by garrensmith** [comment]
```
The following files were not formatted:

  * test/multiple_rows_test.exs
```
Can you run mix format that is causing issues.

**Comment by dottorblaster** [comment]
@garrensmith 

```
﻿λ Alessio-Biancalana elixir → λ git port-multiple-rows-test-to-elixir → mix format --check-formatted
λ Alessio-Biancalana elixir → λ git port-multiple-rows-test-to-elixir → git status
Sul branch port-multiple-rows-test-to-elixir
Your branch is up to date with 'origin/port-multiple-rows-test-to-elixir'.

nothing to commit, working tree clean
λ Alessio-Biancalana elixir → λ git port-multiple-rows-test-to-elixir →
```

Apparently mix format is not working on my end, I'll give updates once I figure that out.

**Comment by dottorblaster** [comment]
@garrensmith looks like I found the problem, newer versions of Elixir don't check for line length anymore, do you think we should discuss this in an separate issue?

Anyway, up for a review :shipit: 👍

**Comment by garrensmith** [comment]
> newer versions of Elixir don't check for line length anymore, do you think we should discuss this in an separate issue?

Yes lets do that.

File: test/elixir/test/multiple_rows_test.exs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            function(doc) {
+              for (var i = 0; i < doc.cities.length; i++)
+                emit(doc.cities[i] + \", \" + doc._id, null);
+            }
+          """
+        }
+      }
+    }
+
+    design_resp = Couch.put("/#{db_name}/_design/list_cities_and_state", body: design_doc)
```
Comment (garrensmith) [inline test/elixir/test/multiple_rows_test.exs]: Could you add `query: %{w: 3}` to this put so we know that design doc is written to each node.

File: test/elixir/test/multiple_rows_test.exs [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    assert design_resp.status_code === 201
```
Comment (garrensmith) [inline test/elixir/test/multiple_rows_test.exs]: Could you change this to:
`assert resp.status_code in [201, 202]`

File: test/elixir/test/multiple_rows_test.exs (hunk 1 above)
Comment (dottorblaster) [reply test/elixir/test/multiple_rows_test.exs]: Sure, you're right

**Pull request merged**